jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -71,6 +75,12 @@
  */
 public class FailoverHttpClient {
 
+  private static enum Failover {
```

Comment:
maybe a little comment explaining what this enum is for? Is this exclusively for the failover history?